src/main/resources/static/js/angular-parish-file/src/app/family/family.component.css

```diff
@@ -0,0 +1,149 @@
+#new-family-form {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  margin-left: 2em;
+}
+
+#update-family-form {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  margin-left: 2em;
+}
+
+.button {
+  max-width: 110px;
+  text-align: center;
+}
+
+input {
+  margin-bottom: 0.5em;
+  margin-left: 1em;
+  border: none;
+  border-bottom: 1px solid thistle;
+}
+
+label {
+  padding: 5px;
+}
+
+table {
+  margin-top: 1em;
+width: 100%;
+  border-collapse: collapse;
+
+}
+
+thead {
+  border-bottom: 1px solid black;
+  border-top: 1px solid black;
+  font-family: regular, sans-serif;
+}
+
+th {
+  padding: 1em;
+}
+
+tbody tr:hover {
+  background-color: thistle !important;
+  cursor: pointer;
+}
+
+table tbody tr:nth-child(odd) {
+  background-color: #F9F5F9;
+}
+
+input:focus {
+  outline: none;
+}
+
+#test {
+  position: relative;
+  width: 80%;
+  max-width: 1400px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#new-family-button {
+}
+
+.button {
+  font-size: 1em;
+  padding: 10px;
+  color: mediumorchid;
+  border: 2px solid mediumorchid;
+  border-radius: 10px;
+  text-decoration: none;
+  cursor: pointer;
+  transition: all 0.3s ease-out;
+  background-color: white;
+  font-weight: lighter;
+}
+
+.button:hover {
+  background: #fdf4fc;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: rgba(0, 0, 0, 0.7);
+  transition: opacity 500ms;
+  visibility: hidden;
+  opacity: 0;
+}
+.overlay:target {
```

Comment:
as this is sass you can inherit target eg:

.overlay {
position: fixed;
......

:target {
...